### PR TITLE
Make configurable via environment variables

### PR DESCRIPTION
**Pull request by andrewcole**
Adds the option to pass the configuration flags via environment variables.

### CODE REVIEW

File: cmd/leader-elector/main.go [hunk 1]
```diff
@@ -3,23 +3,25 @@ package main
...
 	"context"
 	"encoding/json"
-	"flag"
 	"net/http"
 	"os"
 	"os/signal"
 	"syscall"
 	"time"
 
+  "github.com/alexflint/go-arg"
```
Comment (kkosmrli) [inline cmd/leader-elector/main.go]: Indentation is crooked here :)

**Review by kkosmrli — CHANGES_REQUESTED** [review]
Thanks for your contribution. I like the env option a lot, good idea 👍  Did not know `go-arg` before but it looks good and seems slim enough.

We just have some conflicts after the recent changes.

File: cmd/leader-elector/main.go (hunk 1 above)
Comment (andrewcole) [reply cmd/leader-elector/main.go]: Was due to my editor using spaces not tabs - I think I've fixed it all now!

**Comment by andrewcole** [comment]
Rebased to latest master, updated changes to include environment variables for new config options.

**Review by kkosmrli — APPROVED** [review]
Looks good, thank you! 🥇

**Pull request merged**